runtime/themes/solarized_dark.toml

```diff
@@ -73,18 +73,20 @@
 "error" = { fg = "red", modifiers= ["bold", "underlined"] }
 "info" = { fg = "blue", modifiers= ["bold", "underlined"] }
 "hint" = { fg = "base01", modifiers= ["bold", "underlined"] }
-"diagnostic" = { mdifiers = ["underlined"] }
+"diagnostic" = { modifiers = ["underlined"] }
```

Comment:
Can you check and fix the light theme too? For example https://github.com/helix-editor/helix/blob/ee93de5c1a08016507694b6123eaaa77672d709b/runtime/themes/solarized_light.toml#L76 is broken there too

Reply:
I think it's a 1:1 copy with the background color swapped

Reply:
Yep, fixed. I added a screenshot of the fix to the PR description.